resume-buddy/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -112,4 +96,56 @@ private boolean collides(UUID id) {
 
     return results.countEntities(FetchOptions.Builder.withDefaults()) != 0;
   }
+
+  /* Checks if the given user of the given userType has a match of the opposite userType. */
+  public static boolean hasMatch(String userType, String email) {
```

Comment:
This function should take a `UserType` enum value instead of a string, so you can only pass in UserType.REVIEWEE or UserType.REVIEWER, instead of any arbitrary string object, to this function.

Convert the enum value to a string when you need it as a string value to query the data store.